scripts/tests/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM owncloudci/nodejs:11.x
```

Comment:
why don't you use the official node image?

Reply:
agreed I should. I'm trying to make the full thing work first and then will move to official log.